data-msgpack-0.0.5 benchmark build failure

As seen on the stackage build server:

```
Preprocessing executable 'msgpack-parser' for data-msgpack-0.0.5...
[1 of 1] Compiling Main             ( tools/msgpack-parser.hs, dist/build/msgpack-parser/msgpack-parser-tmp/Main.o )
Linking dist/build/msgpack-parser/msgpack-parser ...
Preprocessing benchmark 'benchmark' for data-msgpack-0.0.5...
[1 of 1] Compiling Main             ( bench/benchmark.hs, dist/build/benchmark/benchmark-tmp/Main.o )

bench/benchmark.hs:6:1: error:
    Failed to load interface for ‘Data.MessagePack.IntBench’
    Perhaps you meant
      Data.MessagePack.Instances (from data-msgpack-0.0.5)
      Data.MessagePack.Object (from data-msgpack-0.0.5)
      Data.MessagePack.Spec (needs flag -package-key messagepack-0.5.3)
    Use -v to see a list of the files searched for.

bench/benchmark.hs:7:1: error:
    Failed to load interface for ‘Data.MessagePackBench’
    Perhaps you meant
      Data.MessagePack (from data-msgpack-0.0.5)
      Data.MessagePack (needs flag -package-key messagepack-0.5.3)
      Data.MessagePack.Get (from data-msgpack-0.0.5)
    Use -v to see a list of the files searched for.
```

How I reproduced this locally:

``` bash
$ stack unpack data-msgpack-0.0.5
$ cd data-msgpack
$ stack init --resolver nightly-2016-10-24
$ stack build --bench
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,6 +18,6 @@ script:
   - cabal test
 
 after_script:
-  - TESTSUITE=data-msgpack-0.0.6
+  - TESTSUITE=data-msgpack-0.0.8
   - cp dist/hpc/mix/testsuite/*.mix dist/hpc/mix/$TESTSUITE/
   - hpc-coveralls $TESTSUITE

--- a/data-msgpack.cabal
+++ b/data-msgpack.cabal
@@ -1,5 +1,5 @@
 name:                 data-msgpack
-version:              0.0.6
+version:              0.0.8
 synopsis:             A Haskell implementation of MessagePack
 homepage:             http://msgpack.org/
 license:              BSD3

--- a/data-msgpack.cabal
+++ b/data-msgpack.cabal
@@ -18,6 +18,12 @@ description:
   since the original author is unreachable. This fork incorporates a number of
   bugfixes and is actively being developed.
 
+extra-source-files:
+  bench/Data/MessagePack/*.hs
+  bench/Data/*.hs
+  test/Data/MessagePack/*.hs
+  test/Data/*.hs
+
 source-repository head
   type:             git
   location:         https://github.com/TokTok/hs-msgpack.git

